src/ui/buttons.html

```diff
@@ -6,9 +6,10 @@
 <html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
 <head>
     <meta charset="utf-8" />
-    <title>Button Men Online &mdash; Buttons</title>
+    <title>Button Men Online</title>
```

Comment:
Whatever shows up in buttons.html will be replaced by the JavaScript (unless an error occurs reading from the API). I found having the title flash from "Button Men Online" to "something -- Button Men Online" less jarring than flashing from "Buttons -- Button Men Online" to "something -- Button Men Online". So this was intentional, but I'm happy to change it. (Try it first!)